lib/active_model/serializer/adapter/attributes.rb

```diff
@@ -2,6 +2,11 @@ module ActiveModel
   class Serializer
     module Adapter
       class Attributes < Base
+        def initialize(serializer, options = {})
+          super
+          @include_tree = IncludeTree.from_include_args(options[:include] || '*')
```

Comment:
Could you be more specific? If you're referring to doing `@include_tree = IncludeTree.from_include_args(options[:include]) || '*'`, then it wouldn't work as 1. `#from_include_args` always returns an empty `IncludeTree` for "shady" inputs, and 2. even if that was not the case, we would still have to parse the `'*'`.